styles.css

```diff
@@ -112,15 +113,51 @@ body {
   opacity: 1;
 }
 
-.info {
-  display: flex;
-  justify-content: space-between;
+.main-content {
+  padding: 0 63% 200px 0;
+  background: top right no-repeat url(img/image.svg);
+  margin-bottom: 133px;
+}
+
+.main-content__title {
+  margin: 0 0 40px 0;
+  font: bold 66px/87px 'PT Serif', Arial, Helvetica, sans-serif;
+}
+
+.main-content__title_highlight {
+  background: #ECF3FF;
```

Comment:
Codacy found an issue: [Expected "#ECF3FF" to be "#ecf3ff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/kastykina_app.creation/pullRequest?prid=6278926)